project-base/docker/nginx/nginx.conf

```diff
@@ -42,7 +42,7 @@ server {
         fastcgi_param SCRIPT_FILENAME $realpath_root/resolveFriendlyUrl.php;
     }
 
-    location ~ ^\/(admin|public|content(-test)?|ckeditor|build|bundles|graphql|_profiler|_wdt|file|redirect|elfinder|efconnect|personal-overview-export|codeception-acceptance-test|_error) {
+    location ~ ^\/(admin|public|content(-test)?|ckeditor|build|bundles|graphql|_profiler|_wdt|file|redirect|elfinder|efconnect|personal-overview-export|codeception-acceptance-test|_error|social-network) {
```

Comment:
https://github.com/shopsys/deployment/blob/main/kubernetes/configmap/nginx.yaml needs to be edited in the same manner. 

Then, you need to release a new version of the deployment package and require it in your composer.json
We can discuss this as well :wink:

Reply:
added https://github.com/shopsys/deployment/pull/17